middle/units-extension.owl

```diff
@@ -112,7 +112,6 @@ email: emanuele.ghedini@unibo.it</rdfs:comment>
 
```

Comment:
Changes in units-extension.owl will most likely create many not-easy-to-resolve conflicts with PR #93. I will suggest to exclude changes to this file in this PR. PR #93 already includes the lang attribute to natural language annotations.